README.md

```diff
@@ -5,40 +5,191 @@ Please edit this README.md file with your answers to the worksheet questions.
+|false|false|false|false               |
+|false|false|true |false               |
+|false|true |false|false               |
+|false|true |true |false               |
+|true |false|false|false               |
+|true |false|true |false               |
+|true |true |false|true                |
+|true |true |true |false               |
+
 
 ### b
+|A    |B    |C    |A AND NOT (B AND NOT C)  |
+|-----|-----|-----|-------------------------|
+|false|false|false|false                    |
+|false|false|true |false                    |
+|false|true |false|false                    |
+|false|true |true |false                    |
+|true |false|false|true                     |
+|true |false|true |true                     |
+|true |true |false|false                    |
+|true |true |true |true                     |
 
 ### c
+|A    |B    |C    |(A OR NOT B) AND (A OR C) |
+|-----|-----|-----|--------------------------|
+|false|false|false|false                     |
+|false|false|true |true                      |
+|false|true |false|false                     |
+|false|true |true |false                     |
+|true |false|false|true                      |
+|true |false|true |true                      |
+|true |true |false|true                      |
+|true |true |true |true                      |
 
 ### d
+|A    |B    |C    |D    |A AND NOT (B OR NOT C) AND (NOT A AND D) |
+|-----|-----|-----|-----|-----------------------------------------|
+|false|false|false|false|false                                    |
+|false|false|false|true |false                                    |
+|false|false|true |false|false                                    |
+|false|false|true |true |false                                    |
+|false|true |false|false|false                                    |
+|false|true |false|true |false                                    |
+|false|true |true |false|false                                    |
+|false|true |true |true |false                                    |
+|true |false|false|false|false                                    |
+|true |false|false|true |false                                    |
+|true |false|true |false|false                                    |
+|true |false|true |true |false                                    |
+|true |true |false|false|false                                    |
+|true |true |false|true |false                                    |
+|true |true |true |false|false                                    |
+|true |true |true |true |false                                    |
 
 ## Question 2
 
 ### a
+![](https://github.com/JBroughton2/comp110-worksheet-4/blob/master/Logic%20Circuit%201.PNG)
 
 ### b
+![](https://github.com/JBroughton2/comp110-worksheet-4/blob/master/Logic%20Circuit%202.PNG)
 
 ### c
+![](https://github.com/JBroughton2/comp110-worksheet-4/blob/master/Logic%20Circuit%203.PNG)
 
 ### d
+![](https://github.com/JBroughton2/comp110-worksheet-4/blob/master/Logic%20Circuit%204.PNG)
```

Comment:
All correct -- however it would be helpful to label the inputs (especially since they are not in order in parts c and d)